packages/docs/src/page-configs/ui-elements/stepper/examples/Custom.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <va-stepper
+    v-model="step"
+    :steps="steps"
+    color="danger"
+    noControls
+  >
+    <template #divider>
+      💸💸💸
+    </template>
+
+    <template
+      v-for="step, i in steps"
+      :key="step.label"
+      #[`step-${i}`]="{ setStep, isActive }"
```

Comment:
```suggestion
      #[`button-${i}`]="{ setStep, isActive }"
```

Reply:
I remade them to 'step-button' and 'step-content', just button isn't clear enough imo.